### PR TITLE
changed the way tests are triggered nightly on poc cluster

### PR DESCRIPTION
**Pull request by pawelpaszki**
## What
https://issues.jboss.org/browse/INTLY-1821

## Verification
Compare with recently merged pds-testing-nightly-trigger

### CODE REVIEW

**Comment by pawelpaszki** [comment]
@trepel @tremes can you guys please have a look?

File: jobs/integr8ly/poc-testing-executor.yaml [hunk 1]
```diff
@@ -195,28 +226,30 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
...
+                string(name: 'EVALS_USERNAME', value: "${EVALS_USERNAME}"),
+                string(name: 'EVALS_PASSWORD', value: "${EVALS_PASSWORD}"),
                 string(name: 'ADMIN_USERNAME', value: "${OC_USER}"),
                 string(name: 'ADMIN_PASSWORD', value: "${OC_PASSWORD}"),
-                string(name: 'NAMESPACE_PREFIX', value: "${NAMESPACE_PREFIX}")
+                string(name: 'CUSTOMER_ADMIN_USERNAME', value: "${CUSTOMER_ADMIN_USERNAME}"),
+                string(name: 'CUSTOMER_ADMIN_PASSWORD', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+                string(name: 'RELEASE_VERSION', value: "${RELEASE_VERSION}"),
+                string(name: 'NAMESPACE_PREFIX', value: "${NAMESPACE_PREFIX}"),
+                string(name: 'WEBAPP_URL', value: "https://tutorial-web-app-${NAMESPACE_PREFIX}webapp.apps.${YOURCITY}"),
```
Comment (tremes) [inline jobs/integr8ly/poc-testing-executor.yaml]: This doesn't seem to me as a full and correct URL. Is it? I guess something like CLUSTER_HOSTNAME suffix is missing.

Comment (pawelpaszki) [reply jobs/integr8ly/poc-testing-executor.yaml]: good catch. There should be CLUSTER_HOSTNAME instead of YOURCITY

**Comment by pawelpaszki** [comment]
I am merging this one then. Will also update the pipelines on Jenkins

**Pull request merged**